### ISSUE TITLE
Some question about this project

### ISSUE THREAD

**Issue by BEST8OY** [open-spaced-repetition/fsrs4anki#59]
Hello
I wanted to ask some questions about this project of yours.
1- What does this Custom scheduling which is FSRS does? its difference with anki default scheduler? With example please! and in ELI5 mode (Explain like i'm 5 years old)
2- Who are the target community of this project?
Sorry for these noob like questions

**Comment by L-M-Sherlock** [comment]
You can see these two wiki pages about the scheduler.

https://github.com/open-spaced-repetition/fsrs4anki/wiki/Free-Spaced-Repetition-Scheduler

https://github.com/open-spaced-repetition/fsrs4anki/wiki/Compare-Anki's-built-in-scheduler-and-FSRS

**Comment by L-M-Sherlock** [comment]
This project aims at developing open-source spaced repetition algorithm for popular flashcards software.

**Comment by BEST8OY** [comment]
@L-M-Sherlock 
So the addon (FSRS4ANKI) is for rescheduling anki algorithm scheduled cards?
And for using FSRS algorithm for new cards all i should do is just copying fsrs4anki_scheduler.js code in Custom scheduling section of anki 2.1.55?

**Comment by L-M-Sherlock** [comment]
fsrs4anki_scheduler.js code is used to schedule the new cards. FSRS4Anki Helper add-on is used to reschedule the existing cards. And you can use fsrs4anki_optimizer.ipynb to train a set of parameters more adaptive to you.

**Comment by BEST8OY** [comment]
Thank you for the detailed answers.
